index.html

```diff
@@ -0,0 +1,726 @@
+            </div>
+        </nav>
+        <nav class="navbar navbar-dark bg-danger fixed-top px-1">
+            <a href="#" class="navbar-brand pt-1 pb-1"><img class="small" src="./images/logo-white.png" alt="logo"></a>
+            <div class="navbar-nav flex-row ml-auto mt-1">
+                <a class="nav-item nav-link text-white font-weight-bold pr-1 pr-sm-1" href="#">SIGN IN</a>
+                <a class="nav-item nav-link bg-white text-danger font-weight-bold px-1 px-sm-1" href="#">SUBSCRIBE</a>
+            </div>
+            <div class="navbar-nav">
+                <button class="navbar-toggler px-1" type="button" data-toggle="collapse" data-target="#navbarNav-scroll">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+            </div>
+            <div class="collapse navbar-collapse" id="navbarNav-scroll">
+                <ul class="navbar-nav align-items-stretch bg-white">
+                    <li class="nav-item order-1">
+                        <a class="nav-link text-body text-right font-weight-bold" href="#">India</a>
+                    </li>
+                    <li class="nav-item order-1">
+                        <a class="nav-link text-body text-right font-weight-bold" href="#">World</a>
+                    </li>
+                    <li class="nav-item order-1">
+                        <a class="nav-link text-body text-right font-weight-bold" href="#">Business</a>
+                    </li>
+                    <li class="nav-item order-1">
+                        <a class="nav-link text-body text-right font-weight-bold" href="#">Tech & Science</a>
+                    </li>
+                    <li class="nav-item order-1">
+                        <a class="nav-link text-body text-right font-weight-bold" href="#">Culture</a>
+                    </li>
+                    <li class="nav-item order-1">
+                        <a class="nav-link text-body text-right font-weight-bold" href="#">Sports</a>
+                    </li>
+                    <li class="nav-item order-1">
+                        <a class="nav-link text-body text-right font-weight-bold" href="#">Health</a>
+                    </li>
+                    <li class="nav-item order-1">
+                        <a class="nav-link text-body text-right font-weight-bold" href="#">Opinion</a>
+                    </li>
+                    <li class="nav-item order-1">
+                        <a class="nav-link text-body text-right font-weight-bold" href="#">Ndelible</a>
+                    </li>
+                    <li class="nav-item order-0">
+                        <form class="form-inline" method="GET" action="./search-app">
+                            <input class="ml-auto form-control" type="search" placeholder="&#128269;">
+                        </form>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <div class="container bg-white">
```

Comment:
Change that part, please

```html
<main>
    <div class="row-fluid">
        <div class=""></div>
        <div class=""></div>
    </div>
</main>
````

Reply:
I don't feel comfortable making this change, because the responsive behaviour of my page is already correct. I have checked my page multiple times (both locally and online), and the responsive behaviour is correct for all different screen width breakpoints.

If you see something that is wrong, point it out to me and then I will make the change.